src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/ContentDbContextModelSnapshot.cs

```diff
@@ -3721,7 +3721,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                             b1.HasKey("PublicationId");
 
-                            b1.ToTable("Publications");
+                            b1.ToTable("ExternalMethodology");
```

Comment:
@jamescutts Could you look into this one?

Reply:
This is just mapping the table that external methodology belongs to. To get round the limitations of the ORM its now mapped to its own table rather than the existing publications one.

Reply:
@benoutram or @duncan-at-hiveit